WeakAurasOptions/LoadOptions.lua

```diff
@@ -898,7 +898,7 @@ function OptionsPrivate.ConstructOptions(prototype, data, startorder, triggernum
       type = "input",
       width = WeakAuras.normalWidth,
       name = L["Count"],
-      desc = L["Occurence of the event, reset when aura is unloaded\nCan be a range of values\nCan include repetition using cron notation\nCan have multiple values separated by a comma or a space\n\nExemples:\n2nd 5th and 6th events: 2 5 6\n2nd to 6th: 2-6\nevery 2 events: /2\nevery 3 events starting from 2nd: 2/3\nevery 3 events starting from 2nd and ending at 11th: 2-11/3"],
+      desc = L["Occurence of the event, reset when aura is unloaded\nCan be a range of values\nCan have multiple values separated by a comma or a space\n\nExamples:\n2nd 5th and 6th events: 2, 5, 6\n2nd to 6th: 2-6\nevery 2 events: /2\nevery 3 events starting from 2nd: 2/3\nevery 3 events starting from 2nd and ending at 11th: 2-11/3"],
```

Comment:
I removed the mention of "cron" here, because the syntax is close but not entirely identical to cron.
And cron is most likely not something that people are familar with.

Reply:
We could/should link to the wiki here